addons/website_slides/views/website_slides_templates_course.xml

```diff
@@ -168,110 +168,109 @@
             </t>
 
             <div class="o_wslides_course_main">
-                <!-- ========== TRAINING COURSE ========== -->
-                <div t-if="channel.channel_type == 'training'" class="container">
-                    <div class="row">
-                        <!-- Training Sidebar -->
-                        <div class="col-12 col-md-4 col-lg-3 mt-3 mt-md-0">
-                            <t t-call="website_slides.course_sidebar"/>
-                        </div>
+                <div class="container mb-5">
 
-                        <!-- Training Content -->
-                        <div class="col-12 col-md-8 col-lg-9">
-                            <ul class="nav nav-tabs o_wslides_nav_tabs flex-nowrap" role="tablist" id="profile_extra_info_tablist">
+                    <!-- Row top tabs -->
+                    <div class="row">
+                        <div class="col-12 col-md-8 offset-md-4 offset-lg-3 col-lg-9">
+                            <ul class="nav nav-tabs o_wslides_nav_tabs o_wslides_doc_nav_tabs flex-nowrap" role="tablist" id="profile_extra_info_tablist">
                                 <li class="nav-item">
                                     <a t-att-class="'nav-link %s' % ('active' if active_tab == 'home' else '')"
                                         id="home-tab" data-toggle="pill" href="#home" role="tab" aria-controls="home"
                                         t-att-aria-selected="'true' if active_tab == 'home' else 'false'">
-                                        <i class="fa fa-home"/> Course
+                                        <i class="fa fa-home"/>
+                                        Course
                                     </a>
                                 </li>
-                                <li t-if="channel.allow_comment" class="nav-item o_wslides_course_header_nav_review_training">
+                                <li t-if="channel.allow_comment" class="nav-item o_wslides_course_header_nav_review_documentation">
```

Comment:
I would rename it `o_wslides_course_header_nav_review` as it now works for both types